milestone_project_1_original.py

```diff
@@ -0,0 +1,114 @@
+"""
+A simple Tic-Tac-Toe game.
+"""
+
+BOARD = [' ', ' ', ' ',
+         ' ', ' ', ' ',
+         ' ', ' ', ' ']
+
+CURRENT_PLAYER = "X"
+
+GAME_ON = True
+
+WINNER = " "
+
+
+def display_board():
+    """Display board"""
+    print("Game board      Position Guide")
+    print("  " + BOARD[0] + "|" + BOARD[1] + "|" + BOARD[2] + "             1|2|3")
+    print("  -----             -----")
+    print("  " + BOARD[3] + "|" + BOARD[4] + "|" + BOARD[5] + "             4|5|6")
+    print("  -----             -----")
+    print("  " + BOARD[6] + "|" + BOARD[7] + "|" + BOARD[8] + "             7|8|9")
+
+
+def handle_turn():
+    """Ask for player's input and place the mark on the board"""
+    print(CURRENT_PLAYER + "'s turn!")
+    position = int(input("Choose a position from 1-9: "))
+
+    position_valid = False
+
+    while not position_valid:

```

Comment:
this while loop could be replaced by try-except statememt to get the valid int from user input

Reply:
Tried using Try Except in fb0f154a4f4b349e5f4df36efcb36a118d9b7e6d